SD_DB_Setup.sql

```diff
@@ -1,4 +1,5 @@
-drop database SisterDistrict_dev;
+SET SQL_MODE='ALLOW_INVALID_DATES';
+drop database if exists SisterDistrict_dev;
 
```

Comment:
@LinneaHarts has fixed the date format for the expenditure date column, is that why this is here? if not, wanted to double-check this change.